main/location.conf.template

```diff
@@ -54,10 +54,12 @@ location / {
   proxy_busy_buffers_size 24k;
   proxy_buffers 64 4k;
 
-  proxy_set_header Host $proxy_host;
+  proxy_set_header Host $AEM_ADDR;
```

Comment:
@hlbraddock @cru-Luis-Rodriguez Should it only set this when proxying to AEM and not WordPress?

Reply:
Yes

Reply:
ok, I'll work on that after I get staging ECS multi-container running. It'll be something we need to do in lua.